index.html

```diff
@@ -4,12 +4,30 @@
   <title>The Cheese Guardian</title>
   <meta charset='UTF-8'/>
   <link rel="stylesheet" href="main.css">
+  <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Open+Sans" />
 </head>
 <body>
   <div class="main">
     <img class="cellphone" src="img/phone.png" alt="cellphone">
     <hr>
     <img class="logo" src="img/tcg_logo.png" alt="logo">
   </div>
+  <div class="features">
+    <div class="feature">
+      <img src="img/ic_list.png" alt="">
```

Comment:
try always adding something to alt. Maybe list here and reminder and safe on the others